message.go

```diff
@@ -11,63 +11,69 @@ func (c *Client) getMessageBaseEndpoint() string {
 	return fmt.Sprintf("https://%s/api/%s", c.cfg.BaseURL, c.cfg.Version)
 }
 
+// MessageHeadRequest head request object.
 type MessageHeadRequest struct {
 	ChannelId string `json:"channelid"`
 }
 
-type MessageHeadReply struct {
-}
-
+// MessageWriteRequest request struct for creating a new message.
 type MessageWriteRequest struct {
 	ChannelId string `json:"channelid"`
 	Message   string `json:"message"`
 }
 
+// MessageWriteReply is the object returned when creating a new message.
```

Comment:
Is there a url we could add at the top of this file that describes each of these messages in detail, from the original spec or something?